### DISCUSSION TITLE
Can you max out Lifmunk Effigy?

### DISCUSSION THREAD

**Discussion by NotShocked** [oMaN-Rod/palworld-save-pal#30] [Q&A] [answered]
I am trying to figure out how to max out just capture power from Lifmunk effigies. Or give myself enough I can just buy it.

I tried putting 999 of them in key items. Which show up in game as a separate thing than the ones you find in the world. You cannot upgrade with them.

Is this something possible to do currently?

**Comment by oMaN-Rod** [comment, marked as answer]
Not yet, it's a planned feature though. Lifmunk effigies requires editing the player sav files, which would also open up a bunch of other edits.
